vite.config.ts

```diff
@@ -20,6 +20,10 @@ export default defineConfig({
       exclude: ['**/*.test.ts'],
       outDir: 'dist',
       insertTypesEntry: true,
+      copyDtsFiles: true,
+      beforeWriteFile(filePath, content) {
+        return filePath.match(/\/(vite|jest)\./) ? false : { filePath, content };
+      },
```

Comment:
This parts just removes the types created for the config files to keep things clean. They're not needed for anything